cvmfs/util/posix.cc

```diff
@@ -37,13 +37,16 @@
 #include <cassert>
 #include <cstdio>
 #include <cstring>
+#include <map>
+#include <set>
 #include <string>
+#include <vector>
 
 #include "fs_traversal.h"
 #include "logging.h"
 #include "platform.h"
 
-using namespace std;  // NOLINT
+//using namespace std;  // NOLINT
```

Comment:
Interesting.  So the ``std`` namespace caused trouble?

Reply:
If I remember correctly it broke compilation on my Macbook, for some reason.

Given the short function names with hight risk of collision, maybe we should try to avoid "using namespace std;"